go_pprof_scraper/README.md

```diff
@@ -0,0 +1,67 @@
+# Agent Check: Go-pprof-scraper
+
+## Overview
+
+This check collects profiles from Go applications which expose the [`/debug/pprof`][1] endpoint.
+
+**NOTE**: Prefer instrumenting services with the [dd-trace-go][12] profiling client library. The client library
+offers better integration with other Datadog services, such as through [code hotspots and enpoint filtering][13].
+This integration should be used for services for which you do not control the source code.
+
+**NOTE**: Using this integration will result in billing for hosts under Datadog's [Continuous Profiler][10] service.
+For more on Continuous Profiler billing, consult our [billing documentation][11].
+
+## Setup
+
+Follow the instructions below to install and configure this check for an Agent running on a host. For containerized environments, see the [Autodiscovery Integration Templates][3] for guidance on applying these instructions.
+
+### Installation
+
+TODO: update this
```

Comment:
What does this need to be for an integration in `integration-extras`?

Reply:
Something like that: https://github.com/DataDog/integrations-extras/tree/master/redisenterprise#installation

The only difference between a -core and an -extras integration is that -extras integrations need to be installed manually

Reply:
Got it, thanks! I updated the installation guide to follow that example.